app/code/Magento/Translation/view/adminhtml/requirejs-config.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+var config = {
+    map: {
+        '*': {
+            mageTranslationDictionary: 'Magento_Translation/js/mage-translation-dictionary'
```

Comment:
Just found that we have similar line in the frontend, maybe we need to move it to base, so it will be available both for frontend & admin?
https://github.com/magento/magento2/blob/71c781df5536a2ec79c9cf42ade0c262396767a3/app/code/Magento/Translation/view/frontend/requirejs-config.js#L12

Reply:
I find it a great idea!

Reply:
Done